channels/google-actions/src/main/kotlin/com/justai/jaicf/channel/googleactions/ActionsTypeToken.kt

```diff
@@ -0,0 +1,12 @@
+package com.justai.jaicf.channel.googleactions
+
+import com.justai.jaicf.channel.googleactions.dialogflow.ActionsDialogflowActivatorContext
+import com.justai.jaicf.generic.ChannelTypeToken
+import com.justai.jaicf.generic.ContextTypeToken
+
+typealias ActionsTypeToken = ChannelTypeToken<ActionsBotRequest, ActionsReactions>
+
+val googleActions: ActionsTypeToken = ChannelTypeToken()
```

Comment:
`actions` looks good with an explicit receiver, like `request.actions` or `reactions.actions`
But without the receiver it might be confusing:
```
action {
  actions {
  
  }
}
```
or
```
action(actions) {

}
```

For me, it looks confusing, but not critical, actually. 
So I can rename this if you strongly suggest.

Reply:
I suggest to pick the same name for both in this case. `actions` or `googleActions`